hypervisor/start.c

```diff
@@ -163,15 +184,28 @@ void drawing_mem_map()
 		putfont8_string(vram, xsize, 150, pos, COL8_FFFFFF, font.Bitmap,
 				buf);
 		if (memmap->type == 1) {
+            free_memory[i][0] = memmap->base;
+            free_memory[i][1] = memmap->length;
+            memtest(memmap->base, memmap->base + memmap->length - 1);
```

Comment:
memtest(memmap->base, memmap->length - 1);  looks better.

compute 'memmap->base + memmap->length -1' in function.

Reply:
the function need put the end address, not the length.
unsigned int memtest(unsigned int start, unsigned int end);

Reply:
will update the memtest api interface.